CHANGELOG.md

```diff
@@ -58,6 +56,8 @@
   `features.field_presence = LEGACY_REQUIRED` in Editions files. This new rule is not active by
   default in existing `v1` and `v1beta1` configurations, for backwards-compatibility reasons.
   Migrate your config to `v2` to use them.
+- Add `buf beta registry archive` and `buf beta registry unarchive` commands for managing
+  labels on the BSR.
```

Comment:
```suggestion
  archivable assets (like labels) on the BSR.
```
The command is not strictly associated with labels.

Reply:
Yeah -- I debated about how to present the docs, the command is not specific to labels, but labels are the only thing that you can interact with, and it's a required flag. I didn't want "archive" to be misconstrued with "deprecate", for example, but maybe I'm digging too deep ^^"

Reply:
While we have structured this command to be a generic archive command, but today we can only archive labels so I think it is clearer to users if we document it as a command for archiving labels. If/when we introduce other resource types we can update it then.

I would make a slight tweak to this wording to say this isn't for "managing" labels, but for archiving and unarchiving labels.